src/main/java/duke/Duke.java

```diff
@@ -0,0 +1,54 @@
+package duke;
+
+import java.util.Scanner;
+import java.io.IOException;
+
+/**
+ * Duke class provides the functionality for Duke chatbot.
+ *
+ * @author Rosa Kang
+ */
```

Comment:
Maybe you can use the `/** header_comment_here*/` structure for the one-line header comments?